README.md

```diff
@@ -94,6 +94,272 @@ The enviroCar App is licensed under the [GNU General Public License, Version 3](
 |Short Term Trim 1      |%   	|   	|   	|   	|
 
 
+# Vosk Model Adaptation
+
+## Compile Kaldi
+
+> A pre-requisite for model adaptation
+
+Kaldi features a standard GNU Makefile based build system. Such a build system is traditionally achieved using the command `make`. The recipe to build a package is in a `Makefile`.
+
+Windows installation guide can be found [here](https://github.com/kaldi-asr/kaldi/blob/master/windows/INSTALL.md).
+We will discuss linux compilation and installation of kaldi here:
+
+### Download Kaldi
+Download and open the kaldi repository
+```sh
+git clone https://github.com/kaldi-asr/kaldi.git
+cd kaldi
+```
+
+**Compiling Kaldi involves compilation of the following two folders `/tools` ad `/src`.**
+
+#### 1. Compile `/tools`
+
+- Enter `/tools`
+    ```sh
+    cd tools
+    ```
+
+- Check the pre-requisites for kaldi
+    ```sh
+    ./extras/check_dependencies.sh
+    ```
+    See if there are any system-level installations you need to do. Check the output carefully. There are some things that will make your life a lot easier if you fix them at this stage.<br>
+    In `extras/`, there are also various scripts to install extra bits and pieces that are used by individual example scripts.  If an example script needs you to run one of those scripts, it will tell you what to do.
+<br>
+- Run `make`
+    ```sh
+    make -j 4
+    ```
+
+    You can do a parallel build by supplying the **"-j"** option to make, e.g. to use 4 CPUs -> -j 4, increase or decrease this according to your system specifications.
+
+    It will take a long time if done with single CPU.
+
+#### 2. Compile `/src`
+
+You must first have completed the installation steps in `../tools/INSTALL` *(compiling OpenFst; getting ATLAS and CLAPACK headers)*.
+
+- Configure
+    ```sh
+    ./configure --shared
+    ```
+
+- Run `make depend`
+    ```sh
+    make depend -j 8
+    ```
+
+- Run `make`
+    ```sh
+    make -j 8
+    ```
+
+And Done 🎉,
+You can go ahead with the next steps.
+
+## Update Vosk Model
+There are many issues which can lead to bad accuracy from model mismatches to software bugs. See [accuracy](https://alphacephei.com/vosk/accuracy) guide for more detailed information on how to debug the accuracy problems. Once you figured out the reason is in the model mismatch you can try to update existing models to get better performance.
```

Comment:
Be careful with copy&pasting whole sections from another documentation. You'd rather consider a reqrite or add appropriate credits.

Reply:
I've noticed several other sections that have been copy and pasted. Please, revise your whole documentation. If sections does not provide any new instructions than the original one from the [Vosk homepage](https://alphacephei.com/vosk/adaptation), please remove these sections and just link to the original guide.